src/pages/authors/AuthorsForm.vue

```diff
@@ -28,14 +31,61 @@ export default {
     }
   },
 
+  computed: {
+    isAuthorsCreate () {
+      return this.$route.name === 'AuthorsCreate'
+    },
+
+    submitTitleLabel () {
+      return this.isAuthorsCreate ? 'Criação de Autores' : 'Edição de Autores'
```

Comment:
poderia ser `${this.isAuthorsCreate ? 'criação': 'edição'} de autores`

Reply:
feito